app/feedperson/utils.py

```diff
@@ -19,6 +19,13 @@ def load_feed_people():
   for person in people:
     location = person.location.fordisp.text
     if (re.match('114 Western Ave', location) or re.match('150 Western Ave', location)):
+      
+      # This regex is a temp fix whilst the location data is inconsistent in PeopleSoft.
+      # Should be removed once the data is fixed.
+      if re.match('150 Western Ave', location):
+        location = re.sub(r'Sci&Eng', 'SEC', location)
+        location = re.sub(r'150 Western Avenue', '150 Western Ave, SEC', location)
```

Comment:
I think that in order to be more tolerant of possibilities in spacing, casing, and/or abbreviation in the pattern that we are searching for, we could have:

```
# This allows for using the word "and" instead of &, using "Science" instead of "Sci"
# as well as "Engineering" instead of "Eng", accidental single/multiple spaces before and after the & or "and", and is case insensitive 
location = re.sub(r'\bSci(?:ence)\s*(?:&|and)\s*Eng(?:ineering)\b', 'SEC', location, flags=re.IGNORECASE)

# This allows for multiple spaces between words, and it allows the pattern to be "150 Western Ave", "150 Western Ave.", or "150 Western Avenue".
# Also, in case the original string already contains "SEC," we could include "SEC" in the pattern so that the result does not have "SEC" twice
# (e.g. "150 Western Ave, SEC, SEC" as the final resulting string)
location = re.sub(r'\b150\s+Western\s+Ave(?:\.|nue)?(?:,\s*SEC)?', "150 Western Ave, SEC", location, flags=re.IGNORECASE)
```

The `\b` also makes sure that "150" for example is at the beginning of the word and would not capture "1150 Western Ave" for instance.

This may be too much for what we are looking for, but I was just thinking through all the possibilities of what data we could be receiving for the locations.

Reply:
Thanks @natalynyu. I've left the first regex as the simple version, as Sci&Eng is the only variant we have for now. I've used your second regex with the slight tweak that I'd like to go with Western Ave. as the standard format, as it matches the 114 folks. Added in 78426e2.